### ISSUE TITLE
building_image_classifier.ipynb

### ISSUE THREAD

**Issue by ksasi** [pytorch/opacus#132]
## 🐛 Bug

Hi,

In building_image_classifier.ipynb example, it appears "optimizer.zero_grad()" is not used in the train function. Please advise if this is intentional and the rationale for the same.

Thanks and Regards,
Sasikanth

**Comment by ffuuugor** [comment]
Hi.
That's indeed intentional. 
We still want to clear gradients, but it happens from within opacus code - as a part of `PrivacyEngine.step()` method.
With the way opacus operates at the moment, any calls to `optimizer.zero_grad()` are redundant.

For more context see pull request which removed the call: #96

**Comment by ksasi** [comment]
Thanks for these details.
